Kernel/GenericInterface/Mapping/XSLT.pm

```diff
@@ -301,10 +302,11 @@ sub Map {
     my $ReturnData = eval {
         $XMLSimple->XMLin(
             $XMLPost,
-            ForceArray => 0,
-            ContentKey => '-content',
-            NoAttr     => 1,
-            KeyAttr    => [],
+            ForceArray    => 0,
+            ContentKey    => '-content',
+            NoAttr        => 1,
+            KeyAttr       => [],
+            SuppressEmpty => '',
```

Comment:
Please add a code comment about the intent? If I understand it correctly, that undef values are turned into empty elements not empty attributes.